docs/LOCAL_DEVELOPMENT.md

```diff
@@ -7,14 +7,16 @@ changes from the command line.
 
 The following dependencies must be installed:
 
-- AWS CLI
+- AWS CLI (v1)
 - Python >=3.9 and pip
 - node.js >= v16.11 and npm >= 8
 - virtualenv
 - Ruby >= 2.6
 - libsnappy-dev/snappy-devel (debian/centos)
 - docker
 - jq
+- Java JRE
+- cfn-lint
```

Comment:
This is in our requirements - my understanding is that when calling an installed pip dependency within venv it will start searching it locally first in `venv/bin`? Perhaps if that's not the case, we should edit the make file to call it using `./venv/bin/cfn-lint` instead?